.github/workflows/smoke.yml

```diff
@@ -186,7 +186,7 @@ jobs:
         gh release download --repo dependabot/cli -p "*linux-amd64.tar.gz"
         tar xzvf *.tar.gz >/dev/null 2>&1
         ./dependabot --version
-        URL=https://api.github.com/repos/dependabot/smoke-tests/contents/tests/smoke-${{ matrix.suite.name }}.yaml
+        URL=https://api.github.com/repos/dependabot/smoke-tests/contents/tests/smoke-${{ matrix.suite.name }}.yaml?ref=use-redirect-dot-github
```

Comment:
This should **not** make it into the default branch. It's pointing to [a branch](https://github.com/dependabot/smoke-tests/pull/40) which holds the expected output of this branch's concerns. I'm open to input re: whether it should remain as part of commit history.

Reply:
☝️ was a proof-of-concept wherein I updated the redirect URL in the smoke-tests repo. This is the resulting build result 👇 
<img width="998" alt="image" src="https://user-images.githubusercontent.com/5278382/223764937-9b373c8f-d831-4c3f-9519-f5662d68b1e3.png">

---

I'll be removing that ref now.